configs/openmrs/initializer_config/ordertypes/order_types.csv

```diff
@@ -2,7 +2,7 @@ Uuid,Void/Retire,Name,Description,Java class name,Parent,Concept Classes
 ef83e09d-8885-4f12-be6b-06a80aa62972,,Product,An order for a product,org.openmrs.Order,,Product
 5338a5b1-2cbc-4081-9a9b-9e479e2acaad,,Radiology Order,An order for radiology,org.openmrs.Order,,Radiology
 16bb6bcf-2e81-4ba5-8583-ff4913201a72,,Drug Order,An order for a medication to be given to the patient,org.openmrs.DrugOrder,,Drug
-f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2,,Lab Order,An order for laboratory tests,org.openmrs.Order,,LabTest;LabSet
-52a447d3-a64a-11e3-9aeb-50e549534c5e,,Test Order,Order type for test orders,org.openmrs.TestOrder,,Test
+f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2,TRUE,Lab Order,An order for laboratory tests,org.openmrs.Order,,LabTest;LabSet
+52a447d3-a64a-11e3-9aeb-50e549534c5e,,Test Order,Order type for test orders,org.openmrs.TestOrder,,Test;LabTest;LabSet
```

Comment:
Thanks for the PR @vasharma05. However I don't think it's possible to assign the same `Concept Class` to more than one `Order Type`. I'd recommend first testing the above change on an instance to confirm creation of retired `f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2` and un-retired `52a447d3-a64a-11e3-9aeb-50e549534c5e`.